src/main/java/core/basesyntax/Calculator.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Calculator {
+    public double calculate(double value1, double value2, Character operation) {
```

Comment:
тут можна і просто "char"
```suggestion
    public double calculate(double value1, double value2, char operation) {
```

Reply:
согласен